### PR TITLE
Updates Personal health care contacts instructions to call HEC

### PR DESCRIPTION
**Pull request by radavis**
## Are you removing, renaming or moving a folder in this PR?

- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

### :warning: TeamSites :warning:

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- Updates contact information for updating personal health care contacts

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#89276

## Testing done

- Locally verified via api mocks

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |     Screenshot 2024-08-02 at 14-43-29 Personal Health Care Contacts Veterans Affairs   |    Screenshot 2024-08-02 at 16-53-32 Personal Health Care Contacts Veterans Affairs   |
| Desktop |   Screenshot 2024-08-02 at 14-43-42 Personal Health Care Contacts Veterans Affairs     |    Screenshot 2024-08-02 at 16-53-19 Personal Health Care Contacts Veterans Affairs   |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

`/profile/contacts`

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by jonathanjnelson** [comment]
Looks good @radavis. Thank you

**Review by carlosfelix2 — DISMISSED** [review]
Tested locally. Approved with suggestions

File: src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx [hunk 1]
```diff
@@ -21,7 +27,11 @@ const Contacts = ({ data }) => {
       ),
     }))
   ) : (
-    <Instructions testId="phcc-no-ecs" contactType="emergency contact" />
+    <Instructions
+      testId="phcc-no-ecs"
+      contactType="emergency contact"
```
[the remark this answers is not in the document]
Comment (radavis) [reply src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx:32]: There's a mismatch between what the API returns as a "contact type" and what we display as the rendered text.

File: src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx [hunk 2]
```diff
@@ -2,24 +2,9 @@ import React from 'react';
...
-const DESCRIPTIONS = {
-  [CONTACT_TYPES[0]]: 'The person we’ll contact in an emergency.',
-  [CONTACT_TYPES[1]]:
-    'The person we’ll contact if your primary contact isn’t available.',
-  [CONTACT_TYPES[2]]:
-    'The person you want to represent your health care wishes if needed.',
-  [CONTACT_TYPES[3]]:
-    'The person you want to represent your health care wishes if your primary contact isn’t available.',
-};
+// modeled after VA Profile's Health Benefit AssociatedPersonBio
```
[the remark this answers is not in the document]
Comment (radavis) [reply src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx:7]: Note that this line wasn't added in this PR, but was moved in this PR.

File: src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx [hunk 3]
```diff
@@ -64,7 +49,7 @@ const Contact = ({
   const ordinal = index === 0 ? 'Primary ' : 'Secondary ';
   const title = capitalize(`${ordinal}${baseContactType}`);
 
-  const description = DESCRIPTIONS[contactType];
+  const description = CONTACT_TYPES[contactType];
```
[the remark this answers is not in the document]
Comment (radavis) [reply src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx:52]: `./constants.js` exports a few different constants in different formats, dependent upon what is needed by the component. `CONTACT_TYPES` is needed, here.

**Review by asiisii — APPROVED** [review]
Tested locally using mock data without contact information
LGTM 🚀

Screenshot 2024-08-06 at 10 08 28 AM

**Pull request merged**